CMakeLists.txt

```diff
@@ -27,21 +27,27 @@ enable_testing()
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+# Suppress the warning related to the new policy on fetch content's timestamp
+if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.24.0")
+      cmake_policy(SET CMP0135 NEW)
+endif()
+
 # Location of current CPU torch
```

Comment:
Incidental: but does this part work on Windows?

Reply:
I've no idea whether it works on windows...